taxcalc/functions.py

```diff
@@ -551,7 +554,11 @@ def ItemDed(e17500_capped, e18400_capped, e18500_capped,
                  ID_StateLocalTax_c[MARS - 1])
     c18500 = min((1. - ID_RealEstate_hc) * e18500_capped,
                  ID_RealEstate_c[MARS - 1])
-    c18300 = c18400 + c18500
+    if ID_StateLocalTax_crt < 1:
+        c18300 = min(c18400 + c18500,
+                     ID_StateLocalTax_crt * max(c00100, 0))
```

Comment:
It looks to me like this line applies the cap to the combination of state and local income and sales taxes _and_ state, local, and foreign real estate taxes. If that is your intent, the description of the parameters should be changed.